cli/src/main/java/io/specmesh/cli/KafkaProvisionCommand.java

```diff
@@ -42,7 +43,7 @@
         description =
                 "Apply the provided specification to provision kafka resources and permissions on"
                         + " the cluster")
-public class KafkaProvisionCommand implements Runnable {
+public class KafkaProvisionCommand implements Callable<Provisioner.Status> {
```

Comment:
Unrelated to this change, but noticed below you've got:

```
names = {"-br", "--bootstrap-server"},
```

...shouldn't the short name be `-bs` not `-br`?

Reply:
oops!